zip-1014.rst

```diff
@@ -144,37 +157,36 @@ address(es) for each block. This Dev Fund will end at the second halving (unless
 extended/modified by a future ZIP).
 
 
-ECC slice (Electric Coin Company)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-This slice of the Dev Fund will flow to ECC.
-
-ECC MUST undertake a firm obligation to use the Dev Fund only in support of the
-Zcash cryptocurrency and its community.
-
-In particular, ECC MUST commit to not distribute the Dev Fund proceeds to its
-partners ("shareholders"), other than:
-
-1. In fair-market-value compensation for specific new work (e.g., to employees
-   and contractors).
-2. For covering pass-through tax obligations to partners caused by ECC's receipt
-   of the Dev Fund.
-
-(ECC is encouraged to transition to a corporate structure that would avoid the
-latter taxes.)
+BP slice (Bootstrap Project)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This obligation MUST be made irrevocable, e.g., within ECC's corporate
-governance structure (i.e., its Operating Agreement) or contractual obligations.
+This slice of the Dev Fund will flow as charitable contributions from
+the Zcash Community to the Bootstrap Project, the newly formed parent
+organization to the Electric Coin Company. The Bootstrap Project is organized
+for exempt educational, charitable, and scientific purposes in
+compliance with Section 501(c)(3) of the Code, including but not
+limited to furthering education, information, resources, advocacy,
+support, community, and research relating to cryptocurrency and
+privacy, including Zcash. This slice will be used at the discretion of
+the Bootstrap Project for any purpose within its mandate to support financial
+privacy and the Zcash platform as permitted under Section 501(c)(3). The
+BP slice will be treated as a charitable contribution from the
+Community to support these educational, charitable, and scientific
+purposes.
```

Comment:
It certainly seems consistent with the intent of the original wording to me.

Reply:
The issue would be whether the funds can be used to support financial privacy other than specifically in support of "the Zcash cryptocurrency and its community". If this is a change intended by both ECC and ZF, then no problem

Reply:
My belief is that the intent of the original wording has not changed, so any Zcash DF coins received by Bootstrap are still intended solely for improving Zcash. Although Boostrap's mission can have a larger scope than ZIP 1014 DF funding, it would be surprising and deserve discussion for the organizational change (transferring ECC from previous owners to Bootstrap) somehow changed the intended use of funds.